camera/src/definitions.ts

```diff
@@ -0,0 +1,197 @@
+export type CameraPermissionState = PermissionState | 'limited';
+
+export type CameraPermissionType = 'camera' | 'photos';
+
+export interface CameraPermissionStatus {
+  camera: CameraPermissionState;
+  photos: CameraPermissionState;
+}
+
+export interface CameraPluginPermissions {
+  permissions: CameraPermissionType[];
+}
+
+export interface CameraPlugin {
+  /**
+   * Prompt the user to pick a photo from an album, or take a new photo
+   * with the camera.
+   *
+   * @since 1.0.0
+   */
+  getPhoto(options: CameraOptions): Promise<CameraPhoto>;
+
+  /**
+   * Check camera and photo album permissions
+   *
+   * @since 1.0.0
+   */
+  checkPermissions(): Promise<CameraPermissionStatus>;
+
+  /**
+   * Request camera and photo album permissions
+   *
+   * @since 1.0.0
+   */
+  requestPermissions(
+    permissions: CameraPluginPermissions | null,
```

Comment:
```suggestion
    permissions?: CameraPluginPermissions
```
I think it should be like this, so `Camera.requestPermissions();` can be called without params instead of passing an empty array